code/modules/reagents/chemistry/reagents/impure_reagents/impure_medicine_reagents.dm

```diff
@@ -0,0 +1,412 @@
+	ph = 2.1
+
+/datum/reagent/peptides_failed/on_mob_life(mob/living/carbon/owner, delta_time, times_fired)
+	owner.adjustFireLoss(-0.5 * delta_time)
+	owner.adjustBruteLoss(-1.5* delta_time)
+	owner.adjust_nutrition(-5 * REAGENTS_METABOLISM * delta_time)
+	. = ..()
+
+////Lenturi
+
+//impure
+/datum/reagent/impurity/lentslurri //Okay maybe I should outsource names for these
+	name = "Lentslurri"//This is a really bad name please replace
+	description = "A highly addicitive muscle relaxant that is made when Lenturi reactions go wrong."
+	addiction_types = list(/datum/addiction/medicine = 8)
+	liver_damage = 0
+
+/datum/reagent/impurity/lentslurri/on_mob_metabolize(mob/living/carbon/owner)
+	owner.add_movespeed_modifier(/datum/movespeed_modifier/reagent/lenturi)
+	return
+
+/datum/reagent/impurity/lentslurri/on_mob_end_metabolize(mob/living/carbon/owner)
+	owner.remove_movespeed_modifier(/datum/movespeed_modifier/reagent/lenturi)
+	return
+
+//failed
+/datum/reagent/inverse/ichiyuri
+	name = "Ichiyuri"
+	description = "Prolonged exposure to this chemical can cause an overwhelming urge to itch oneself."
+	reagent_state = LIQUID
+	color = "#C8A5DC"
+	ph = 1.7
+	addiction_types = list(/datum/addiction/medicine = 2.5)
+	tox_damage = 0.1
+	///Probability of scratch - increases as a function of time
+	var/resetting_probability = 0
+	///Prevents message spam
+	var/spammer = 0
+
+//Just the removed itching mechanism - omage to it's origins.
+/datum/reagent/inverse/ichiyuri/on_mob_life(mob/living/carbon/owner, delta_time, times_fired)
+	if(prob(resetting_probability) && !(HAS_TRAIT(owner, TRAIT_RESTRAINED) || owner.incapacitated()))
+		if(spammer < world.time)
+			to_chat(owner,"<span class='warning'>You can't help but itch yourself.</span>")
+			spammer = world.time + (10 SECONDS)
+		var/scab = rand(1,7)
+		owner.adjustBruteLoss(scab*REM)
+		owner.bleed(scab)
+		resetting_probability = 0
+	resetting_probability += (5*(current_cycle/10) * delta_time) // 10 iterations = >51% to itch
+	..()
+	return TRUE
+
+////Aiuri
+
+//impure
+/datum/reagent/impurity/aiuri
+	name = "Aivime"
+	description = "This reagent is known to interfere with the eyesight of a patient."
+	ph = 3.1
+	addiction_types = list(/datum/addiction/medicine = 1.5)
+	liver_damage = 0.1
+	//blurriness at the start of taking the med
+	var/cached_blurriness
+
+/datum/reagent/impurity/aiuri/on_mob_add(mob/living/owner, amount)
+	. = ..()
+	cached_blurriness = owner.eye_blurry
+	owner.set_blurriness(((creation_purity*10)*(volume/metabolization_rate)) + cached_blurriness)
+
+/datum/reagent/impurity/aiuri/on_mob_delete(mob/living/owner, amount)
+	. = ..()
+	if(owner.eye_blurry <= cached_blurriness)
+		return
+	owner.set_blurriness(cached_blurriness)
+
+////Hercuri
+
+//inverse
+/datum/reagent/inverse/hercuri
+	name = "Herignis"
+	description = "This reagent causes a dramatic raise in a patient's body temperature."
```

Comment:
gunpowder syringe users are making the jerma sus face right now

Reply:
wait, is this one of the ones that can only be created by an ingestion reaction?

Reply:
It's not ingestion only - but that's a good point, I'll make it so that it's ingestion only. When 15%< to <32% it'll 100% convert into this when added to a mob.

Reply:
Done